index.bs

```diff
@@ -5818,7 +5816,7 @@ can be set to [=value of string literal tokens|the value=] of a
 <h4 oldids="dom-ByteString" id="idl-ByteString" interface>ByteString</h4>
 
 The {{ByteString}} type
-corresponds to the set of all possible sequences of bytes.
+corresponds to [=byte sequences=].
 Such sequences might be interpreted as UTF-8 encoded strings [[!RFC3629]]
 or strings in some other 8-bit-per-code-unit encoding, although this is not required.
```

Comment:
I wonder if we should remove this. In particular because when something returns a ByteString it's not expected that would not be treated as an isomorphic string. (Isomorphic string is indeed not defined at the moment, but perhaps at some point we should add it to Infra as the equivalent to the 8-bit strings implementations have when ASCII string isn't sufficient.)

Okay to leave this for a follow-up though.

Reply:
Filed #1295 on this.